app/add_some_abilities/add_potion_abilities.py

```diff
@@ -0,0 +1,9 @@
+def add_ability_from_potion(knight: dict, ability: str) -> None:
+    if check_presence_ability(knight, ability):
+        knight[ability] += knight["potion"]["effect"].get(ability)
+
+
+def check_presence_ability(knight: dict, ability: str) -> bool:
+    if ability in knight["potion"]["effect"]:
```

Comment:
The function `check_presence_ability` should check if the ability is present in the knight's abilities, not in `knight['potion']['effect']`. This logic needs to be corrected.